From dfe32357e526027c9e17a5e3527d04557bd2cedc Mon Sep 17 00:00:00 2001
From: David Bauer <mail@david-bauer.net>
Date: Sun, 14 Oct 2018 19:43:47 +0200
Subject: [PATCH] tools: tplink-safeloader: add C7v5 entries

This adds more entries to the SupportList fpr the Archer C7 v5.

This fixes issues with flash from vendor web-ui.

Signed-off-by: David Bauer <mail@david-bauer.net>
---
 tools/firmware-utils/src/tplink-safeloader.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/tools/firmware-utils/src/tplink-safeloader.c b/tools/firmware-utils/src/tplink-safeloader.c
index b2cc96bc3c..4b227e1c97 100644
--- a/tools/firmware-utils/src/tplink-safeloader.c
+++ b/tools/firmware-utils/src/tplink-safeloader.c
@@ -626,7 +626,11 @@ static struct device_info boards[] = {
 		.support_list =
 			"SupportList:\n"
 			"{product_name:Archer C7,product_ver:5.0.0,special_id:00000000}\n"
-			"{product_name:Archer C7,product_ver:5.0.0,special_id:55530000}\n",
+			"{product_name:Archer C7,product_ver:5.0.0,special_id:45550000}\n"
+			"{product_name:Archer C7,product_ver:5.0.0,special_id:55530000}\n"
+			"{product_name:Archer C7,product_ver:5.0.0,special_id:43410000}\n"
+			"{product_name:Archer C7,product_ver:5.0.0,special_id:4A500000}\n"
+			"{product_name:Archer C7,product_ver:5.0.0,special_id:54570000}\n",
 
 		.support_trail = '\x00',
 		.soft_ver = "soft_ver:1.0.0\n",
-- 
2.19.1

